bytes/json.go

```diff
@@ -146,9 +146,10 @@ func normalizeBytes(s []byte) []byte { //nolint:gocyclo // It's OK.
 			default:
```

Comment:
How does the unquoteBytes function differ from normalizeBytes. It's not obvious. Do we need both of them? Maybe add a comment about it?

Reply:
An appropriate comment was added to both functions.